mne/baseline.py

```diff
@@ -24,7 +24,7 @@ def _log_rescale(baseline, mode='mean'):
 
 
 @verbose
-def rescale(data, times, baseline, mode='mean', copy=True, verbose=None):
+def rescale(data, times, baseline, mode='mean', copy=None, verbose=None):
```

Comment:
Actually never mind, I meant to revert changes to functions and only change
methods. I'll fix it.